### PR TITLE
Add Ukrainian translation for `docs/uk/docs/external-links.md`

### PR DESCRIPTION
**Pull request by mezgoodle**
Translate the page into ukrainian language

### CODE REVIEW

**Review by dimaqq — APPROVED** [review]

**Comment by mezgoodle** [comment]
Are there some problems with the pr? It was reviewed in October last year, but still didn't merged

File: docs/uk/docs/external-links.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+* <a href="{{ article.link }}" class="external-link" target="_blank">{{ article.title }}</a> by <a href="{{ article.author_link }}" class="external-link" target="_blank">{{ article.author }}</a>.
+{% endfor %}
+{% endif %}
+
+### Японською
+
+{% if external_links %}
+{% for article in external_links.articles.japanese %}
+
+* <a href="{{ article.link }}" class="external-link" target="_blank">{{ article.title }}</a> by <a href="{{ article.author_link }}" class="external-link" target="_blank">{{ article.author }}</a>.
```
Comment (dimaqq) [inline docs/uk/docs/external-links.md:28]: I was wondering if "title **by** author" should be translated... 
Seems to be a convention though, see e.g. Polish https://fastapi.tiangolo.com/pl/external-links/

**Review by dimaqq — COMMENTED** [review]
@tiangolo poke, should be ready to go!

File: docs/uk/docs/external-links.md (hunk 1 above)
Comment (ArcLightSlavik) [reply docs/uk/docs/external-links.md:28]: Probably wouldn't hurt

**Review by ArcLightSlavik — APPROVED** [review]

**Comment by ArcLightSlavik** [comment]
@tiangolo
 gif

**Pull request closed without merging**